docs/building/windows-instructions.md

```diff
@@ -5,10 +5,11 @@ You can build ML.NET either via the command line or by using Visual Studio.
 
 ## Required Software
 
-1. **[Visual Studio 2017](https://www.visualstudio.com/downloads/) (Community, Professional, Enterprise)**.  The Community version is completely free.
+1. **[Visual Studio 2017](https://www.visualstudio.com/downloads/) (Community, Professional, Enterprise)** version 18.5.4 or higher. The Community version is completely free.
```

Comment:
>18.5.4 [](start = 115, length = 6)

15.8.4 maybe?
And I have 15.8.1 (I don't update VS often) and it works as well! :)